rest/project-generator/src/main/java/io/syndesis/project/converter/visitor/StepVisitorContext.java

```diff
@@ -29,6 +29,8 @@
 @JsonDeserialize(builder = StepVisitorContext.Builder.class)
 public interface StepVisitorContext extends Iterator<StepVisitorContext> {
 
+    GeneratorContext getGeneratorContext();
+
```

Comment:
Isn't it required that the generatorContext is also propagated in the `next()` step ?

Reply:
Next does something like `new StepVisitorContext.Builder().createFrom(this)` which copies the generator context too

Reply:
ok, got it. thx ;-)